api/serialization.go

```diff
@@ -1,18 +1,9 @@
 package api
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
 	"io/ioutil"
-	"net/http"
-	"path/filepath"
-	"strings"
 
 	"github.com/qri-io/dataset"
-	"github.com/qri-io/dataset/detect"
-	"github.com/qri-io/qri/base/fill"
-	"gopkg.in/yaml.v2"
 )
 
 // inlineScriptsToBytes consumes all open script files for dataset components
```

Comment:
very little left of this file after moving the `formFileDataset` func, let's move `inlineScriptToBytes` into `api/dataset.go`, the only place it's currently used